src/java/org/apache/cassandra/cql3/conditions/ColumnCondition.java

```diff
@@ -364,6 +502,11 @@ public ElementAccessBound(ColumnMetadata column,
                                    List<ByteBuffer> values)
         {
             super(column, operator);
+            checkForUnsetValues(values);
+
+            boolean isMap = column.type instanceof MapType;
```

Comment:
nit: might be worth inlining so we don't have to do anything if the element isn't `null`

Reply:
Inlined.

Optimizer probably already does the inlining, but who knows.